### ISSUE TITLE
The default lr_scheduler in the code and the lr_scheduler in the paper are different.

### ISSUE THREAD

**Issue by Heibaizhu** [FadeoN/Distilling-Image-Dehazing-With-Heterogeneous-Task-Imitation#3]
Hello,
You've done a amazing job! But I have a few small questions about this code.
I found that the LR_scheduler(ReduceLROnPlateau) in the code you provided was inconsistent with the Lr_scheduler(cyclincal cosine) in the paper. Does that mean that you finally found ReduceLROnPlateau better?  Which one is more in line with the accuracy in your paper?  Can you provide the CyclicLR parameters such as base_lr, max_lr,and decay_rate? 
Looking forward to your reply!
Best Regards
